debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -11,20 +11,14 @@
 import java.util.OptionalLong;
 import java.util.concurrent.atomic.AtomicReference;
 
+import io.debezium.connector.postgresql.connection.*;
```

Comment:
Could we move this near the other io.debezium.connector.postgresql imports ?
Also why do we want to introduce * imports. Let us retain it in the way it was